alerts/alerts.css

```diff
@@ -0,0 +1,25 @@
+.alert {
+  display: block;
+  text-align: center;
+  width: 15rem;
+  padding: 1rem;
+  border-radius: 0.5rem;
+  font-weight: lighter;
+  margin: 1rem;
+}
+
+.alert-primary {
+  background-color: var(--sky-blue);
```

Comment:
If these colors are common to all the components, please add them to a common stylesheet. If not, looks good!

Reply:
Actually, these colors are coming from the main stylesheet `styles.css`. Those are defined in the root styling section itself. I mean all the colors or anything which is shared among various markups is defined into a common main stylesheet i.e. `styles.css` in the root directory.